games/applications/grovers.py

```diff
@@ -250,7 +249,7 @@ def grovers_dist(num_qubits, marked_item, n_iterations):
 @pytest.mark.parametrize("optimization_level", [0, 1, 2, 3])
 @pytest.mark.parametrize("backend", backends)
 def bench_qiskit_grovers(benchmark, optimization_level, backend):
-    ### This code is what is used to benchmark grover's search algorithm
+    """This code is what is used to benchmark grover's search algorithm"""
     shots = 70000
     expected_counts = {format(14, "b").zfill(5): shots}
```

Comment:
Please change this line so you can get valid fidelity.
```suggestion
expected_counts = {format(SECRET_STATE, "b").zfill(QUBIT_COUNT): shots}
```